demo/full/styles/style.css

```diff
@@ -1,30 +1,35 @@
-body {
-  margin: 0px;
+  @font-face {
+  font-family: "icons";
+  src: url("../fonts/FontAwesome.otf");
+  font-weight: normal;
+  font-style: normal;
 }
 
 @font-face {
-  font-family: "icons";
-  src: url("../fonts/FontAwesome.otf");
+  font-family: "Roboto";
+  src: url("../fonts/Roboto-Regular.otf");
   font-weight: normal;
   font-style: normal;
 }
 
+body {
+  margin: 0px;
+  font-family: "Roboto";
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Unexpected missing generic font family ([css:S4649](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4649))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXOEfCulgbYnohY_20T&open=AWXOEfCulgbYnohY_20T)

[ISSUE_KEY]: AWXOEfCulgbYnohY_20T